fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -171,16 +173,16 @@ def test_create_big_object_and_download_it_from_container(self):
         big_file_url_1 = self.conf[PROPERTIES_CONFIG_TEST][PROPERTIES_CONFIG_SWIFT][PROPERTIES_CONFIG_SWIFT_BIG_FILE_1]
         f = urllib2.urlopen(big_file_url_1)
         data = f.read()
-        with open(SWIFT_RESOURCES_PATH + TEST_BIG_OBJECT_PREFIX + TEST_BIG_FILE_EXTENSION, "wb") as code:
+        with open(SWIFT_RESOURCES_PATH + big_object_name, "wb") as code:
             code.write(data)
 
-        self.test_world['local_objects'].append(TEST_BIG_OBJECT_PREFIX + TEST_BIG_FILE_EXTENSION)
+        self.test_world['local_objects'].append(big_object_name)
 
         # Uploading the object
         response = self.swift_operations.create_object(containerName, SWIFT_RESOURCES_PATH
-                                                            + TEST_BIG_OBJECT_PREFIX + TEST_BIG_FILE_EXTENSION,
-                                                            big_object_name)
-        origin = hashlib.md5(open(SWIFT_RESOURCES_PATH + TEST_BIG_OBJECT_PREFIX + TEST_BIG_FILE_EXTENSION, 'rb')
+                                                            + big_object_name,
+                                                            remote_big_object_name)
+        origin = hashlib.md5(open(SWIFT_RESOURCES_PATH + big_object_name, 'rb')
```

Comment:
I would change the name of the variable from origin to originHash and remote to remoteHash or something similar in order to give more semantic to what kind of thing I am manage in this variable.